product_docs/docs/pgd/5/durability/limitations.mdx

```diff
@@ -0,0 +1,109 @@
+
+## General limitations
+
+-   Replacing a node with its physical standby doesn't work for nodes that use
+    CAMO/Eager/Group Commit. We don't recommend combining physical standbys and EDB Postgres
+    Distributed, even if it's possible.
+
+-   [Legacy synchronous replication](legacy-sync) uses a mechanism for transaction confirmation
+    different from the one used by CAMO, Eager, and Group Commit. The two aren't
+    compatible, so don't use them together. Whenever you use Group Commit, CAMO
+    or Eager, make sure none of the PGD nodes are configured in
+    `synchronous_standby_names`.
+
+-   Postgres two-phase commit (2PC) transactions (that is, [`PREPARE
+    TRANSACTION`](https://www.postgresql.org/docs/current/sql-prepare-transaction.html))
+    can't be used with CAMO, Group Commit, or Eager because those
+    features use two-phase commit underneath.
+
+## Group Commit
+
+[Group Commit](group-commit) enables configurable synchronous commits over 
+nodes in a group. If you use this feature, take the following limitations into account:
+
+-   Not all DDL can run when you use Group Commit. If you use unsupported DDL, a warning is logged, and the transactions commit scope is set to local. The only supported DDL operations are:
+    - Nonconcurrent `CREATE INDEX`
+    - Nonconcurrent `DROP INDEX`
+    - Nonconcurrent `REINDEX` of an individual table or index
+    - `CLUSTER` (of a single relation or index only)
+    - `ANALYZE`
+    - `TRUNCATE`
+
+- Explicit two-phase commit is not supported by Group Commit as it already uses two-phase commit.
+
+- Combining different commit decision options in the same transaction or
+  combining different conflict resolution options in the same transaction isn't
+  supported.
+
+- Currently, Raft commit decisions are extremely slow, producing very low TPS.
+  We recommended using them only with the `eager` conflict resolution setting
+  to get the Eager All-Node Replication behavior of PGD 4 and older.
+
+## Eager
+
+[Eager](../consistency/eager) is available through Group Commit. It avoids conflicts by eagerly aborting transactions that might clash. It's subject to the same limitations as Group Commit.
+
+- Eager doesn't allow the `NOTIFY` SQL command or the `pg_notify()` function. It
+  also don't allow `LISTEN` or `UNLISTEN`.
+
+## CAMO
+
+[Commit At Most Once](camo) (CAMO) is a feature that aims to prevent
+applications committing more than once. If you use this feature, take 
+these limitations into account when planning:
+
+- CAMO is designed to query the results of a recently failed COMMIT on the
+origin node. So, in case of disconnection, code the application to immediately
+request the transaction status from the CAMO partner. Have as little delay as
+possible after the failure before requesting the status. Applications must not
+rely on CAMO decisions being stored for longer than 15 minutes.
```

Comment:
```suggestion
origin node. In case of a connection failure, the application must
request the transaction status from the CAMO partner immediately. Ensure to have as little delay as
possible after the failure before requesting the status. Applications must not
rely on CAMO decisions being stored for longer than 15 minutes.
```